package.json

```diff
@@ -55,8 +55,7 @@
   },
   "optionalDependencies": {
     "@a4u/a4u-collection-essential-ui-icons-release": "^4.1.0",
```

Comment:
While we're at it, can we get a pubic package for these little fellas too?

Reply:
Yeah, we should create a ticket for next sprint.